src/app/modules/item/http-services/get-item-children.service.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+export interface ItemChild {
+  string: {
+    title: string,
```

Comment:
The title is nullable. Always read carefully the doc to set the type.

Reply:
fixed